examples/cloud-router-2-aws-connection/main.tf

```diff
@@ -0,0 +1,37 @@
+terraform {
+  required_providers {
+    equinix = {
+      source  = "equinix/equinix"
+      version = ""
+    }
+  }
+}
+
+provider "equinix" {
+  client_id     = var.equinix_client_id
+  client_secret = var.equinix_client_secret
+}
+
+module "cloud_router_aws_connection" {
+  source = "../../modules/cloud-router-connection"
+
+  connection_name       = var.connection_name
+  connection_type       = var.connection_type
+  notifications_type    = var.notifications_type
+  notifications_emails  = var.notifications_emails
+  additional_info       = [{ key = "accessKey", value = var.aws_access_key }, { key = "secretKey", value = var.aws_secret_key }]
```

Comment:
nit: I would put the entire map into the variables file with an `additional_info` variable and then add that variable here. 

No need to change now. For future reference when we get in here next time.